Make.inc

```diff
@@ -888,6 +893,7 @@ endif
 ifeq ($(OS), FreeBSD)
 JLDFLAGS := -Wl,-Bdynamic
 OSLIBS += -lelf -lkvm -lrt -Wl,--export-dynamic -Wl,--version-script=$(JULIAHOME)/src/julia.expmap $(NO_WHOLE_ARCHIVE) $(LIBUNWIND)
+USE_SYSTEM_CURL := 1
```

Comment:
I thought `curl` wasn't installed by default on FreeBSD? I could have sworn I had to `pkg install curl` on my VM, but I could be mistaken.